app/models/answer.rb

```diff
@@ -1,6 +1,7 @@
 class Answer < ApplicationRecord
   belongs_to :user
   belongs_to :question
+  has_one :best_question, class_name: "Question", inverse_of: :best_answer, foreign_key: :best_answer_id
```

Comment:
Looks cool.  Can we sit down at some point and go over this? I've never seen the 'inverse_of' statement before.